process-document/src/main/kotlin/com/ritense/processdocument/domain/impl/delegate/DocumentDelegateImpl.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.processdocument.domain.impl.delegate
+
+import com.ritense.document.service.DocumentService
+import com.ritense.processdocument.domain.impl.CamundaProcessInstanceId
+import com.ritense.processdocument.service.ProcessDocumentService
+import com.ritense.valtimo.contract.authentication.UserManagementService
+import mu.KotlinLogging
+import org.camunda.bpm.engine.delegate.DelegateExecution
+
+class DocumentDelegateImpl(
```

Comment:
Why did you choose to call this a Delegate? It's unusual in this codebase and not very clear about what this does. Also called Impl when it doesn't implement anything?

Reply:
Its because this module, the process-document module, already has 2 of such classes that are intended to be used exclusively for bpmn's:
- DocumentVariableDelegate
- ProcessDocumentStartEventMessageDelegate

This class shouldn't have the `Impl` postfix, so i'll remove that